.dep-versions

```diff
@@ -3,4 +3,7 @@ jax=0.4.23
 mhlo=4611968a5f6818e6bdfb82217b9e836e0400bba9
 llvm=cd9a641613eddf25d4b25eaa96b2c393d401d42c
 enzyme=1beb98b51442d50652eaa3ffb9574f4720d611f1
+
+# Always remove custom PL/LQ versions before release.
 pennylane=f638bc53a09724a83d9d58964bf37bfd438e6aa3
+lightning=0.35.0-dev17
```

Comment:
Let's build against `dev21` to be consistent with the master branch and v0.35.0 of Lightning,

```suggestion
lightning=0.35.0-dev21
```